.github/workflows/start-goth.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+echo CLEANUP DOCKER
+docker rm --force $(docker ps -a -q)
+sudo systemctl restart docker
+
+echo LOGIN TO GHCR.IO
+echo $GITHUB_API_TOKEN | docker login ghcr.io -u $GITHUB_ACTOR --password-stdin
+
+echo REMOVING PREBIOUS GOTH SESSION FILES
+rm /tmp/goth_interactive.env
+rm -rf /root/.cache/
+rm -rf /tmp/goth-tests/
+
+echo CREATING VENV
+rm -rf .envs/goth
+python -m venv .envs/goth
+source .envs/goth/bin/activate
+
+echo INSTALLING TOOLS
+python -m pip install --upgrade pip
+python -m pip install --upgrade setuptools wheel
+
+echo INSTALLING DEPENDENCIES
+python -m pip install goth==0.15.11 pytest pytest-asyncio pexpect
```

Comment:
Maybe we can move the goth version to params so it is easier to upgrade (or even run on multiple versions) later?